google/cloud/forseti/common/util/email/email_factory.py

```diff
@@ -17,13 +17,15 @@
 from builtins import object
 from google.cloud.forseti.common.util import logger
 from google.cloud.forseti.common.util.email import sendgrid_connector
+from google.cloud.forseti.common.util.email import mailjet_connector
```

Comment:
try/except put into `mailjet_connector.py` because this is the place we import `mailjet_rest` client:

```
try:
    from mailjet_rest import Client
    MAILJET_ENABLED = True
except ImportError:
    LOGGER.warning('Cannot enable Mailjet connector because the '
                   '`mailjet_rest` library was not found. Run '
                   '`sudo pip3 install mailjet_rest` to install '
                   'Mailjet.')
    MAILJET_ENABLED = False
```